Makefile

```diff
@@ -79,6 +79,10 @@ backend-config-validation:
 yarn-install: yarn-ensure
 	$(YARN) --cwd frontend install --frozen-lockfile 
 
+.PHONY: backend-ensure-envoy
```

Comment:
@mcutalo88 We talked briefly about pulling in a binary for integration testing before the holidays, lmk if you have thoughts on this approach. I still need to figure out where to put this binary to avoid a permission denied, but figured I'd flag this for you in case you had opinions before I spent more time on this